doc/lightning-listtransactions.7.md

```diff
@@ -0,0 +1,100 @@
+lightning-listtransactions -- Command to get the list of transactions that was stored in the wallet
+============================================================
+
+SYNOPSIS
+--------
+
+**listtransactions**
+
+DESCRIPTION
+-----------
+
+The **listtransactions** command returns transactions tracked in the wallet. This includes deposits, withdrawals and transactions related to channels. A transaction may have multiple types, e.g., a transaction may both be a close and a deposit if it closes the channel and returns funds to the wallet.
+
+EXAMPLE JSON REQUEST
+------------
+```json
+{
+  "id": 82,
+  "method": "listtransactions",
+  "params": {}
+}
+```
+
+RETURN VALUE
+------------
+
+On success, the command will return a list of transactions, each object rappresent the transaction with all details.
+
+- *hash*: A string that rappresent  the hash of transaction, with the caller can use to find it on blockchain.
+- *rawtx*: A string that rappresent the hexdecimal of transaction.
+- *blockheight*: An integer that rappresent the block height that contains the transaction on blockchain.
+- *txindex*: An integer that rappresent the transaction index inside the block.
+- *locktime*: An integer that rappresent the locktime.
+- *version*: An integer that rappresent the version.
+- *inputs*: A list of transactions, each transaction is rappresented with an object with the following proprieties:
```

Comment:
s/A list of transactions, each transaction/A list of spent transaction outputs, each spent transaction output/

s/proprieties/properties/